sources/packages/web/src/services/http/ApplicationOfferingChangeRequestApi.ts

```diff
@@ -126,6 +129,34 @@ export class ApplicationOfferingChangeRequestApi extends HttpBaseClient {
     );
   }
 
+  /**
+   * Gets the Application Offering Change Request details for the student.
+   * @param applicationOfferingChangeRequestId the Application Offering Change Request id.
+   * @returns Application Offering Change Request details.
+   */
+  async getApplicationOfferingDetailsById(
```

Comment:
https://github.com/bcgov/SIMS/pull/2196#discussion_r1297824609

Reply:
@ann-aot I looked up the above comment and found that it points to a comment that is not relevant here. Can you please check once.